ngraph/test/CMakeLists.txt

```diff
@@ -448,7 +448,6 @@ set(MULTI_TEST_SRC
     backend/floor.in.cpp
     backend/floor_mod.in.cpp
     backend/function_name.in.cpp
-    backend/fused_op.in.cpp
```

Comment:
I think you also should delete `backend/fused_op.in.cpp` file

Reply:
Ok, as the tests migration was a part of ops revision,
 just quick check to ensure all operations from `fused_op.in.cpp` have existing tests: 

HardSigmoid tests -> [ngraph/test/backend/hard_sigmoid.in.cpp](https://github.com/mitruska/openvino/blob/mitruska/fusedop_class_removal/ngraph/test/backend/hard_sigmoid.in.cpp)
MVN tests -> [docs/template_plugin/tests/functional/op_reference/mvn.cpp](https://github.com/mitruska/openvino/blob/mitruska/fusedop_class_removal/docs/template_plugin/tests/functional/op_reference/mvn.cpp)
SquaredDifference tests -> [ngraph/test/backend/squared_difference.in.cpp](https://github.com/mitruska/openvino/blob/mitruska/fusedop_class_removal/ngraph/test/backend/squared_difference.in.cpp)
SpaceToDepth tests ->[ngraph/test/backend/space_to_depth.in.cpp](https://github.com/mitruska/openvino/blob/mitruska/fusedop_class_removal/ngraph/test/backend/space_to_depth.in.cpp)

Seems can be removed.

Reply:
In any case you have removed this file from CMake script, it means that we exclude it from the build

Reply:
Sure, done.